fail("Mensaje de error y una variable de facter ${::variable}")
Devuelve un mensaje de error y termina el programa en ese punto devolviendo cÃ³digo de error (return code = 1)
